source/_drafts/Video-js-6-0-0-RC-0-The-first-RC.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Video.js 6.0.0-RC.0: The first Release Candidate'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - video.js 6.0
+---
+
+## The first Release Candidate for 6.0 has been released
+Last week, we began wrapping up months of effort to make Video.js even better with the first Release Candidate (RC) of Video.js 6.0. To make it even better, however, we had to make a [few breaking changes](https://github.com/videojs/video.js/compare/v5.16.0...v6.0.0-RC.0).
```

Comment:
The second sentence reads a bit weird when we say "to make video.js event better" in the first sentence. Perhaps a minor change to:
~~To make it event better~~ In order to make it better